webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -28,4 +48,146 @@ def calculate_pauschbetrag(has_pflegegrad=False, disability_degree=None, has_mer
+            choices=[('yes', 'yes'),('no', '')],
+            render_kw={'data_label':  _l('form.lotse.request_pauschbetrag.data_label')},         
+            validators=[InputRequired(_l('validate.input-required'))])
+        
+    @classmethod
+    def get_label(cls, data=None):
+        return ngettext('form.lotse.person_a.request_pauschbetrag.label', 'form.lotse.person_a.request_pauschbetrag.label',
+                        num=get_number_of_users(data))        
+
+    
+    def render(self):
+        props_dict = PauschbetragPersonProps(            
+            step_header={
+                'title': ngettext('form.lotse.person_a.request_pauschbetrag.title', 'form.lotse.person_a.request_pauschbetrag.title',
+                        num=get_number_of_users(self.stored_data))
+            },
+            form={
+                'action': self.render_info.submit_url,
+                'csrf_token': generate_csrf(),
+                'show_overview_button': bool(self.render_info.overview_url),
+            },
+            pauschbetrag=self.get_pauschbetrag(),
+            fields=form_fields_dict(self.render_info.form),
+            prev_url=self.render_info.prev_url
+        ).camelized_dict()
+        
+        
+        # Humps fails to camelize individual letters correctly, so we have to fix it manually.
+        # (A fix exists but hasn't been released at the time of writing: https://github.com/nficano/humps/issues/61)
+        props_dict['fields']['personARequestsPauschbetrag'] = props_dict['fields'].pop('personA_requestsPauschbetrag')
+        
+
+        return render_template('react_component.html',
+                               component='PauschbetragPersonAPage',
+                               props=props_dict,
+                               form=self.render_info.form,
+                               header_title=_('form.lotse.header-title'))
+        
+        
+class StepPauschbetragPersonB(StepPauschbetrag):
+    name = 'person_b_requests_pauschbetrag'
+    section_link = SectionLink('mandatory_data', StepFamilienstand.name, _l('form.lotse.mandatory_data.label'))
+
+    label = _l('form.lotse.person_b.request_pauschbetrag.label')
+    preconditions = [ShowPersonBPrecondition, PersonBHasDisabilityPrecondition]
+        
+    class InputForm(SteuerlotseBaseForm):
+        person_b_requests_pauschbetrag = SelectField(
+            # TODO: This empty string mapping is because of key mapping - the text is within the react component
+            choices=[('yes', 'yes'),('no', '')],
+            render_kw={'data_label':  _l('form.lotse.request_pauschbetrag.data_label')},         
+            validators=[InputRequired(_l('validate.input-required'))])
+    
+    @classmethod
+    def get_label(cls, data):
+        return cls.label
+    
+    def get_overview_value_representation(self, value):
+        if value == 'yes':
+            return self.get_pauschbetrag() + ' ' + _('currency.euro')
+            
+        return None
```

Comment:
If we keep the implementation in `StepPauschbetrag` we shouldn't need this, I think?

Reply:
The implementation is in the StepPauschbetrag? I don't get it sry :(